datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/mutate/util/LabelUtils.java

```diff
@@ -201,6 +201,21 @@ public static boolean isAuthorizedToUpdateTags(@Nonnull QueryContext context, Ur
         orPrivilegeGroups);
   }
 
+  public static boolean isAuthorizedToAssociateTag(@Nonnull QueryContext context, Urn targetUrn) {
+
+    // Decide whether the current principal should be allowed to associate the tag
+    final DisjunctivePrivilegeGroup orPrivilegeGroups = new DisjunctivePrivilegeGroup(ImmutableList.of(
+        new ConjunctivePrivilegeGroup(ImmutableList.of(PoliciesConfig.ASSOCIATE_TAGS_PRIVILEGE.getType()))
```

Comment:
hm I'm a little confused here - so what's the benefit of this new privilege when we have the `EDIT_ENTITY_TAGS_PRIVILEGE` privilege already that seems to be doing type of thing?

Reply:
> hm I'm a little confused here - so what's the benefit of this new privilege when we have the `EDIT_ENTITY_TAGS_PRIVILEGE` privilege already that seems to be doing type of thing?

Please check my response to your other comment on the need for this new privilege